### DISCUSSION TITLE
multisig with hardware keys

### DISCUSSION THREAD

**Discussion by reduxionist** [btcpayserver/btcpayserver#4524] [General]
is this possible?

**Comment by dennisreimann** [comment]
Do you mean signing a multisig transaction with multiple hardware devices via the BTCPay Server Vault?

**Reply by reduxionist** [reply]
Yes, that's what I meant, sorry...

**Comment by NicolasDorier** [comment]
It is already possible.

**Reply by reduxionist** [reply]
Thanks very much!

**Comment by pavlenex** [comment]
https://docs.btcpayserver.org/Multisig/#step-4-send-a-transaction-from-the-multisig-wallet
